addons/event/models/event_mail.py

```diff
@@ -261,15 +261,15 @@ def _refresh_mail_count_done(self):
                     ("scheduler_id", "=", self.id),
                     ("mail_sent", "=", True),
                 ])
-                self.mail_count_done = total_sent
+                scheduler.mail_count_done = total_sent
             elif scheduler.last_registration_id:
                 total_sent = self.env["event.registration"].search_count([
                     ("id", "<=", self.last_registration_id.id),
                     ("event_id", "=", self.event_id.id),
                     ("state", "not in", ["draft", "cancel"]),
                 ])
-                self.mail_count_done = total_sent
-                self.mail_done = total_sent >= self.event_id.seats_taken
+                scheduler.mail_count_done = total_sent
+                scheduler.mail_done = total_sent >= self.event_id.seats_taken
```

Comment:
@tde-banana-odoo hello :wave: 
Not sure but, could it be that you also forgot the `self.event_id.seats_taken` here and the `self` in the domain above?
There are also the `self` in the first `if`